UI/src/view/LinearAssetForm.js

```diff
@@ -115,15 +124,12 @@
            '</footer>';
   }
 
-  function renderLinktoWorkList() {
-    var notRendered = !$('#work-list-link').length;
-    if(notRendered) {
+  var renderLinktoWorkList = function renderLinktoWorkList(layerName) {
       $('#information-content').append('' +
-        '<div class="form form-horizontal">' +
-          '<a id="work-list-link" class="unknown-linear-assets" href="#work-list/speedLimit">Tuntemattomien nopeusrajoitusten lista</a>' +
-        '</div>');
-    }
-  }
+          '<div class="form form-horizontal" data-layer-name="' + layerName + '">' +
+          '<a id="unchecked-links" class="unchecked-linear-assets" href="#work-list/' + layerName + '">Tuntemattomien nopeusrajoitusten lista</a>' +
+          '</div>');
```

Comment:
this text, needs to be changed for "Tarkistamattomien huoltoteiden lista". Should be filtered by layerName